website/static/js/koHelpers.js

```diff
@@ -396,6 +396,41 @@ ko.bindingHandlers.datePicker = {
 };
 
  /**
+ * Bind content of content editable to observable. Looks for maxlength attr
+ * to limit input. If no attr then no limit.
+ * Example:
+ * <div contenteditable="true" data-bind="editableHTML: <observable_name>" maxlength="500"></div>
+ */
+ko.bindingHandlers.editableHTML = {
+    init: function(element, valueAccessor) {
+        var $element = $(element);
+        var initialValue = ko.utils.unwrapObservable(valueAccessor());
+        $element.html(initialValue);
+        $element.on('change input paste keyup blur', function() {
+            var observable = valueAccessor();
+            // limit input to maxlength
+            var charLimit = $element.attr('maxlength');
+            if (charLimit) {
+                if ($element.text().length < charLimit) {
+                    observable($element.html());
+                } else {
+                    $element.html(observable());
```

Comment:
This has the effect of blocking me from pasting long passages of text. I would expect instead for the text to be cut off instead of disallowed entirely.

##### Steps

1. `python -c "print('a'*499, end='')" | pbcopy`
1. paste into comment box

##### Expected

The first 499 bytes are pasted.

##### Actual

Nothing is pasted.